### ISSUE TITLE
Support Multiple Environments

### ISSUE THREAD

**Issue by jvanderaa** [nautobot/nautobot-app-chatops#43]
### Environment
* Nautobot version:  1.0.1
* nautobot-plugin-chatops version:  1.2.0

### Proposed Functionality

Capability to have multiple instances of the chat environment to work with the single bot.

Slack instance 1 --\
                                 Nautobot Chatops Plugin
Slack instance 2 --/

### Use Case

The ability to have dev and production instances of chat to be used with a single instance of Nautobot Chatops Plugin

**Comment by smk4664** [comment]
Can you assign this one to me? I think it is just a matter of extending the Mattermost Token ability to Slack.

**Comment by smk4664** [comment]
This is getting de-prioritized on my backlog.

**Comment by jvanderaa** [comment]
Agreed. We have talked and that makes sense. If anyone wants to take this one on, please let us know here.
